src/config/colors.toml

```diff
@@ -8,10 +8,8 @@
 
 "commit_id" = "blue"
 "change_id" = "magenta"
-"author" = "yellow"
```

Comment:
`jj show` style slightly changed. I think the new output is also fine.

Reply:
Good catch. Yes, the new output seems fine, so I'll just leave it as is. I think it's nice that the timestamp gets colored. I'm not sure if I think we should add back color to the name.